exp-models/addon/models/experiment.js

```diff
@@ -139,6 +139,21 @@ const Experiment = DS.Model.extend(JamModel, {
         maxDays = maxDays || Number.MAX_SAFE_INTEGER;
         return minDays <= age && age <= maxDays;
     },
+    isOldEnough(participant) {
```

Comment:
PR sent to Kim. Will hold this one until she's had a chance to look over proposed changes, and gives the all-clear for both this and the Lookit companion PR.